source/validator/test/SwapValidator.js

```diff
@@ -686,47 +686,44 @@ contract('SwapValidator', async accounts => {
       errorCodes = await validator.checkSwap.call(order, {
         from: bobAddress,
       })
-      equal(errorCodes[0], 4)
-      equal(web3.utils.toUtf8(errorCodes[1][0]), 'SENDER_INVALID_AMOUNT')
-      equal(web3.utils.toUtf8(errorCodes[1][1]), 'SIGNER_INVALID_AMOUNT')
-      equal(web3.utils.toUtf8(errorCodes[1][2]), 'SENDER_TOKEN_KIND_MISMATCH')
-      equal(web3.utils.toUtf8(errorCodes[1][3]), 'SIGNER_TOKEN_KIND_MISMATCH')
+      equal(errorCodes[0], 2)
+      equal(web3.utils.toUtf8(errorCodes[1][0]), 'SENDER_TOKEN_KIND_MISMATCH')
+      equal(web3.utils.toUtf8(errorCodes[1][1]), 'SIGNER_TOKEN_KIND_MISMATCH')
```

Comment:
did we want to use ethers here in the future?

Reply:
Okay to use web3 in tests and ethers otherwise. Can migrate tests at a later date.